app/javascript/mastodon/api/tags.ts

```diff
@@ -0,0 +1,11 @@
+import { apiRequestPost, apiRequestGet } from 'mastodon/api';
+import type { ApiHashtagJSON } from 'mastodon/api_types/tags';
+
+export const apiGet = (tagId: string) =>
```

Comment:
Wouldn't this be clearer as `apiGetTag` or `apiFetchTag`? I do realize that we have some inconsistencies in naming across modules with `Get` VS `Fetch` and omitting or mentioning the class name (I think the only existing cases where the names are really lacking are `apiCreate` and `apiUpdate` in `app/javascript/mastodon/api/lists.ts`).